spring-pulsar-spring-cloud-stream-binder/src/main/java/org/springframework/pulsar/spring/cloud/stream/binder/properties/PulsarConsumerProperties.java

```diff
@@ -41,7 +41,7 @@ public class PulsarConsumerProperties extends ConsumerConfigProperties {
 	@Nullable
 	private Class<?> messageValueType;
 
-	private Integer partitionCount = 1;
+	private int partitionCount = 0;
 
```

Comment:
Wdyt about using null for the default which we can then interpret as no partitions?